framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/Commands/CreateMigrationAndRunMigrator.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Volo.Abp.Cli.Args;
+using Volo.Abp.Cli.Utils;
+using Volo.Abp.DependencyInjection;
+
+namespace Volo.Abp.Cli.Commands
+{
+    public class CreateMigrationAndRunMigrator : IConsoleCommand, ITransientDependency
+    {
+        public virtual async Task ExecuteAsync(CommandLineArgs commandLineArgs)
+        {
+            if (commandLineArgs.Target.IsNullOrEmpty())
+            {
+                throw new CliUsageException(
+                    "DbMigrations folder path is missing!"
+                );
+            }
+
+            var dbMigratorProjectPath = GetDbMigratorProjectPath(commandLineArgs.Target);
+
+            if (dbMigratorProjectPath == null)
+            {
+                throw new Exception("DbMigrator is not found!");
+            }
+
+            var output = CmdHelper.RunCmdAndGetOutput("cd \"" + commandLineArgs.Target + "\" && dotnet ef migrations add Initial -s " + dbMigratorProjectPath);
```

Comment:
Backslash will not work in macOS. use https://docs.microsoft.com/tr-tr/dotnet/api/system.io.path.directoryseparatorchar?view=net-5.0

Reply:
check that if the `commandLineArgs.Target` is in a different drive is it still working? normally it doesn't work and you need to run `cd /d` command. check that /d command works in macOS.

Reply:
> Backslash will not work in macOS. use https://docs.microsoft.com/tr-tr/dotnet/api/system.io.path.directoryseparatorchar?view=net-5.0

Backslash is not a part of the path. It is used as an escape char `\"`.

> check that if the commandLineArgs.Target is in a different drive is it still working? normally it doesn't work and you need to run cd /d command. check that /d command works in macOS.

As i know (and tested), It is guaranteed to be work in same drive.